src/util/storagePool.tsx

```diff
@@ -1,37 +1,45 @@
 import { AbortControllerState, checkDuplicateName } from "util/helpers";
 import { AnyObject, TestFunction } from "yup";
-import { ConfigRowMetadata } from "./configInheritance";
+import { LxdConfigOptionsKeys } from "types/config";
 
-export const cephStoragePoolDefaults: Record<string, string> = {
-  ceph_cluster_name: "ceph",
-  ceph_osd_pg_num: "32",
-  ceph_rbd_clone_copy: "true",
-  ceph_user_name: "admin",
-  ceph_rbd_features: "layering",
-};
-
-export const getCephStoragePoolDefault = (
-  formField: string,
-): ConfigRowMetadata => {
-  if (formField in cephStoragePoolDefaults) {
-    return { value: cephStoragePoolDefaults[formField], source: "LXD" };
-  }
-  return { value: "", source: "LXD" };
-};
-
-const cephStoragePoolFormFieldToPayloadName: Record<string, string> = {
+export const storagePoolFormFieldToPayloadName: Record<string, string> = {
```

Comment:
Renaming this to a generic function name is a little weird for me since the lookup contains only ceph configs. That being said, only ceph storage pools have extra configs like these, so I guess generalising is fine since this lookup could contain configs for other driver types in the future. I'm just a little conflicted since I don't know if that future scenario is plausible looking at all the supported drivers from the lxd documentation.

Reply:
There are plenty of options for [LVM](https://documentation.ubuntu.com/lxd/en/latest/reference/storage_lvm/#storage-pool-configuration) and [ZFS](https://documentation.ubuntu.com/lxd/en/latest/reference/storage_zfs/#storage-pool-configuration), which I'd like to include soonish. So this will extend to include keys for other drivers.

Reply:
aah okay, I missed those, that's perfect then :+1: